src/auslib/web/public/json.py

```diff
@@ -9,15 +9,22 @@
 from auslib.web.public.helpers import AUS, with_transaction
 
 
+def get_aus_metadata_headers(eval_metadata):
+    header_name_metadata_map = {"rule": "Rule-ID", "rule_data_version": "Rule-Data-Version"}
```

Comment:
I think this would be easier to read if it was simply written as:
```
return {
    "Rule-ID": eval_metadata["rule_id"]
...
}
```

I don't see any benefit to the indirection - but please let me know if there's a reason to keep it that I'm missing.

Reply:
A map was used to handle new evaluation metadata entries that is not mapped to a header, so the header name fallsback to the key. But how we will make `eval_metadata` values explicity set to `unknown`, it is safe to directly acces the key.